ci-operator/step-registry/baremetal/lab/agent/install/baremetal-lab-agent-install-commands.sh

```diff
@@ -88,6 +86,17 @@ if [ "${DISCONNECTED}" == "true" ] && [ -f "${SHARED_DIR}/install-config-mirror.
   OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE="$(<"${CLUSTER_PROFILE_DIR}/mirror_registry_url")/${OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE#*/}"
 fi
 
+echo "[INFO] Looking for patches to the install-config.yaml..."
```

Comment:
these patches should happen only after any processing of the install-config is done here. What's requiring you to move them back before the lines below?

Reply:
I had moved this due to the platform.baremetal.vips section. I have now addressed it within the vips configuration file and moved the patches section back to its original location.